package.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "dd-academy",
   "private": true,
+  "type": "module",
```

Comment:
Curious, what was the reason for changing this to a module?

Reply:
I didn't find any other way. 
remark-frontmatter plug-in for mdx is ESM (not CommonJS anymore) so I can't use require in next.config.js
And that threw me the "cannot use import outside module" error when I changed next.config.js

If you know of a way to do that without using that type: module, please teach me! 😂
I don't have that much exp with this

Reply:
Ok that makes sense. There are ways to do it though I’m not sure it’s entirely worth it without making it intrusive. 

Referring this for extra context: https://github.com/vercel/next.js/issues/9607